.github/workflows/IOnIntegration_Production.yml

```diff
@@ -0,0 +1,44 @@
+name: I-On Integration Production
+
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  IMAGE_NAME: i-on-integration
+
+jobs:
+  push_production:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && contains(github.ref, 'tags')
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build image
+        id: production_build_image
+        run: docker build . --tag image
+
+      - name: Log into registry
+        id: production_login_registry
+        run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login docker.pkg.github.com -u ${{ github.actor }} --password-stdin
+
+      - name: Push image
+        id: production_push_image
+        run: |
+          IMAGE_ID=docker.pkg.github.com/${{ github.repository }}/$IMAGE_NAME
+
+          # Strip git ref prefix from version
+          VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
```

Comment:
I prefer having this logic inside the gradle file, so I can test it locally (and eventually reuse it)